src/CommunityToolkit/Xamarin.CommunityToolkit/Views/MediaElement/Android/FormsVideoView.android.cs

```diff
@@ -14,6 +14,7 @@ public class FormsVideoView : VideoView
 		public FormsVideoView(Context context)
 			: base(context)
 		{
+			SetBackgroundColor(global::Android.Graphics.Color.Transparent);
```

Comment:
Hm, usually renderers don't keep any logic in ctors.
Can we move it to OnElementChanged for example?

Reply:
This isn't a renderer but an extended VideoView but it can easily be moved if necessary

Reply:
Oh, you are right :) Sorry
Allright